app/uk/gov/hmrc/apiplatformmicroservice/thirdpartyapplication/domain/models/applications/ApplicationUpdate.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformmicroservice.thirdpartyapplication.domain.models.applications
+
+import enumeratum.{Enum, EnumEntry, PlayJsonEnum}
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.play.json.Union
+
+import java.time.LocalDateTime
+
+sealed trait ActorType extends EnumEntry
+
+object ActorType extends Enum[ActorType] with PlayJsonEnum[ActorType] {
```

Comment:
I don't think this needs to be a) an enum and b) a play json enum - we only put it into json as strings.

Reply:
OK I will try and get rid of this and the union formatter set these values.

Reply:
Done